movie/src/components/footer/Footer.tsx

```diff
@@ -0,0 +1,23 @@
+import { Container } from "@material-ui/core";
+import React from "react";
+import styled from "styled-components";
+import Logo from "../header/Logo/Logo";
+
+const Footer: React.FC = () => {
+  return (
+    <FooterSection>
+      <Container maxWidth="sm">
+        <Logo />
+      </Container>
+    </FooterSection>
+  );
+};
+
+const FooterSection = styled.div`
```

Comment:
you should directly write 'styled(Conainer)' instead of 'styled.div'.
Please, read the docs:
https://styled-components.com/docs/api#styled

Reply:
No, I can`t, because Container called when I need to align section. But FooterSerction needs to colored all width footer

Reply:
changed